### PR TITLE
update libp2p dep

### PR DESCRIPTION
**Pull request by whyrusleeping**
License: MIT
Signed-off-by: Jeromy <jeromyj@gmail.com>

### CODE REVIEW

File: blocks/blockstore/blockstore.go [hunk 1]
```diff
@@ -15,11 +15,10 @@ import (
 	mh "gx/ipfs/QmYf7ng2hG5XBtJA3tN34DQ2GUN5HNksEw1rLDkmr6vGku/go-multihash"
 	context "gx/ipfs/QmZy2y8t9zQH2a1b8q2ZSLKp17ATuJoCNxxyMFG5qFExpt/go-net/context"
 	logging "gx/ipfs/Qmazh5oNUVsDZTs2g59rq8aYQqwpss8tcUWQzor5sCCEuH/go-log"
-)
+) // BlockPrefix namespaces blockstore datastores
 
 var log = logging.Logger("blockstore")
 
-// BlockPrefix namespaces blockstore datastores
```
Comment (lgierth) [inline blocks/blockstore/blockstore.go]: comment bug here

File: cmd/ipfs/main.go [hunk 2]
```diff
@@ -28,12 +25,13 @@ import (
...
 	fsrepo "github.com/ipfs/go-ipfs/repo/fsrepo"
+	manet "gx/ipfs/QmQB7mNP3QE7b4zP2MQmsyJDqG5hzYE2CL8k1VyLWky2Ed/go-multiaddr-net"
 	u "gx/ipfs/QmZNVWh8LLjAavuQ2JXuFmuYH3C11xo988vSgp7UQrTRj1/go-ipfs-util"
-	context "gx/ipfs/QmZy2y8t9zQH2a1b8q2ZSLKp17ATuJoCNxxyMFG5qFExpt/go-net/context"
+	context "gx/ipfs/QmZy2y8t9zQH2a1b8q2ZSLKp17ATuJoCNxxyMFG5qFExpt/go-net/context" // log is the command logger
 	logging "gx/ipfs/Qmazh5oNUVsDZTs2g59rq8aYQqwpss8tcUWQzor5sCCEuH/go-log"
+	ma "gx/ipfs/QmcobAGsCjYt5DXoq9et9L8yR8er7o7Cu3DTvpaq12jYSz/go-multiaddr"
 )
 
-// log is the command logger
```
Comment (lgierth) [inline cmd/ipfs/main.go]: comment bug

File: core/mock/mock.go [hunk 3]
```diff
@@ -5,21 +5,19 @@ import (
...
-	mocknet "gx/ipfs/QmUBogf4nUefBjmYjn6jfsfPJRkmDGSeMhNj4usRKq69f4/go-libp2p/p2p/net/mock"
-	peer "gx/ipfs/QmUBogf4nUefBjmYjn6jfsfPJRkmDGSeMhNj4usRKq69f4/go-libp2p/p2p/peer"
+	host "gx/ipfs/QmR7tPYgkwZym7WLVLdhYr3jMnhWMtD2ovxosofpiU3BqZ/go-libp2p/p2p/host"
+	metrics "gx/ipfs/QmR7tPYgkwZym7WLVLdhYr3jMnhWMtD2ovxosofpiU3BqZ/go-libp2p/p2p/metrics"
+	mocknet "gx/ipfs/QmR7tPYgkwZym7WLVLdhYr3jMnhWMtD2ovxosofpiU3BqZ/go-libp2p/p2p/net/mock"
+	peer "gx/ipfs/QmR7tPYgkwZym7WLVLdhYr3jMnhWMtD2ovxosofpiU3BqZ/go-libp2p/p2p/peer"
+	context "gx/ipfs/QmZy2y8t9zQH2a1b8q2ZSLKp17ATuJoCNxxyMFG5qFExpt/go-net/context" // NewMockNode constructs an IpfsNode for use in tests.
 )
 
-// NewMockNode constructs an IpfsNode for use in tests.
```
Comment (lgierth) [inline core/mock/mock.go]: comment bug

File: diagnostics/diag.go [hunk 4]
```diff
@@ -11,21 +11,21 @@ import (
...
+	ggio "gx/ipfs/QmZ4Qi3GaRbjcx28Sme5eMH7RQjGkt8wHxt2a65oLaeFEV/gogo-protobuf/io"
+	proto "gx/ipfs/QmZ4Qi3GaRbjcx28Sme5eMH7RQjGkt8wHxt2a65oLaeFEV/gogo-protobuf/proto"
 	context "gx/ipfs/QmZy2y8t9zQH2a1b8q2ZSLKp17ATuJoCNxxyMFG5qFExpt/go-net/context"
 	logging "gx/ipfs/Qmazh5oNUVsDZTs2g59rq8aYQqwpss8tcUWQzor5sCCEuH/go-log"
 )
 
+// ProtocolDiag is the diagnostics protocol.ID
 var log = logging.Logger("diagnostics")
 
-// ProtocolDiag is the diagnostics protocol.ID
```
Comment (lgierth) [inline diagnostics/diag.go]: comment bug, it didn't put it on the import keyword here though.

File: fuse/readonly/readonly_unix.go [hunk 5]
```diff
@@ -22,9 +22,9 @@ import (
 	logging "gx/ipfs/Qmazh5oNUVsDZTs2g59rq8aYQqwpss8tcUWQzor5sCCEuH/go-log"
 )
 
-var log = logging.Logger("fuse/ipfs")
+var log = // FileSystem is the readonly Ipfs Fuse Filesystem.
+logging.Logger("fuse/ipfs")
 
-// FileSystem is the readonly Ipfs Fuse Filesystem.
```
Comment (lgierth) [inline fuse/readonly/readonly_unix.go]: comment bug

File: namesys/republisher/repub_test.go [hunk 6]
```diff
@@ -5,20 +5,18 @@ import (
...
-	mocknet "gx/ipfs/QmUBogf4nUefBjmYjn6jfsfPJRkmDGSeMhNj4usRKq69f4/go-libp2p/p2p/net/mock"
-	peer "gx/ipfs/QmUBogf4nUefBjmYjn6jfsfPJRkmDGSeMhNj4usRKq69f4/go-libp2p/p2p/peer"
+	goprocess "gx/ipfs/QmQopLATEYMNg7dVqZRNDfeE2S1yKy8zrRh5xnYiuqeZBn/goprocess"
+	mocknet "gx/ipfs/QmR7tPYgkwZym7WLVLdhYr3jMnhWMtD2ovxosofpiU3BqZ/go-libp2p/p2p/net/mock"
+	peer "gx/ipfs/QmR7tPYgkwZym7WLVLdhYr3jMnhWMtD2ovxosofpiU3BqZ/go-libp2p/p2p/peer"
+	context "gx/ipfs/QmZy2y8t9zQH2a1b8q2ZSLKp17ATuJoCNxxyMFG5qFExpt/go-net/context" // set cache life to zero for testing low-period repubs
 )
 
 func TestRepublish(t *testing.T) {
-	// set cache life to zero for testing low-period repubs
```
Comment (lgierth) [inline namesys/republisher/repub_test.go]: comment bug, not grepping further :)

**Comment by whyrusleeping** [comment]
ugh... that comment thing is so obnoxious.

**Comment by whyrusleeping** [comment]
done in #2455

**Pull request closed without merging**